instrumentation/opentelemetry-instrumentation-elasticsearch/tests/test_elasticsearch.py

```diff
@@ -50,6 +50,24 @@
 # pylint: disable=too-many-public-methods
 
 
+def normalize_arguments(doc_type, body=None):
+    if major_version == 7:
```

Comment:
how do you feel about ifs in tests? I can provide two different implementations of this helper for < 7 and > 7 if it's preferred but I would like to avoid having to specify each test mock return value for each version :sweat:

Reply:
elasticsearch 8 will use the same path

Reply:
I think this is ok.